edwardsTicApp/protocol/edwardsTic.proto

```diff
@@ -3,7 +3,7 @@
 # Serial Line Setup:
 # 9600 baud, 1 stop bit, 8 data bits, No parity, Xon/Xoff
 
-replytimeout = 1000;
+replytimeout = 500;
```

Comment:
Why has this been reduced? Usually this parameter is set against hardware, since we got this IOC from elsewhere we would usually assume the comms parameters are set correctly, at least until we can test against our own hardware.